app/src/main/java/com/kickstarter/viewmodels/ThreadViewModel.kt

```diff
@@ -79,9 +81,36 @@ interface ThreadViewModel {
                     )
                 }
 
-            comment
+            /*
+             * This block should be removed as its meant to test reply to a comment
+             * The block above should show the replies
+             */
+            commentData
```

Comment:
I think you can remove this block and just add it on the Pull Request on the QA section for the rest of us to QA it. Once that's done I'll approve the PR, it seems ok to me 👍

Reply:
done